src/test/java/net/snowflake/ingest/streaming/internal/FlushServiceTest.java

```diff
@@ -422,7 +427,7 @@ public void testGetFilePath() {
 
   @Test
   public void testFlush() throws Exception {
-    TestContext<?> testContext = testContextFactory.create();
+    TestContext<?> testContext = testContextFactory.create(false);
```

Comment:
as suggested in another thread please using JUnit parameters / somehow parameterize this isIcebergMode so that all tests run with isIcebergMode=on/off, instead of only testing the non-iceberg usecases.

feel free to close this thread out so we just have one thread on this!